### PR TITLE
Minor UI refactor

### PR DESCRIPTION
**Pull request by dexamundsen**
- Add util for process.env
- Minor cleanup of imports
- Add errorHandler.tsx for error related fucntions
- Split route objects into a separate file

---

Reason for change: cleaner and easier when auth code is added

### CODE REVIEW

File: ui/src/errorhandler.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import { Button } from "@mui/material";
```
Comment (tjennison-google) [inline ui/src/errorhandler.tsx:1]: This file should be be "errorPage.tsx" to match the component and standard casing.

File: ui/src/environment.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+// Items here should be added to environment.js.template as well.
```
Comment (tjennison-google) [inline ui/src/environment.tsx:1]: Should this file exist?

File: ui/src/environment.tsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+  REACT_APP_USE_EXIT_URL: boolean;
+  REACT_APP_EXIT_URL: string;
+
+  REACT_APP_USE_FAKE_API: string;
+
+  REACT_APP_GET_LOCAL_AUTH_TOKEN: string;
+};
+
+export function getEnvironment(): Environment {
```
Comment (tjennison-google) [inline ui/src/environment.tsx:16]: What is the context for this change? Particularly considering the comment about `environment.js.template`, I'm having trouble evaluating it.

**Review by tjennison-google — APPROVED** [review]

**Review by marikomedlock — COMMENTED** [review]
LGTM, but will let @tjennison-google approve.

File: ui/src/errorhandler.tsx (hunk 1 above)
Comment (dexamundsen) [reply ui/src/errorhandler.tsx]: renamed

File: ui/src/environment.tsx (hunk 2 above)
Comment (dexamundsen) [reply ui/src/environment.tsx:1]: Removed comment, had copied from VWB-ui which may not be applicable here

File: ui/src/environment.tsx (hunk 3 above)
Comment (dexamundsen) [reply ui/src/environment.tsx:16]: environment.js.template may not b applicable here. removed

**Pull request merged**